website/templates/nav.mako

```diff
@@ -24,7 +24,7 @@
     <div id="navbar" class="navbar-collapse collapse navbar-right">
       <ul class="nav navbar-nav">
         % if user_name:
-            <li id="osfNavDashboard"><a href="/">Dashboard</a></li>
+            <li id="osfNavDashboard"><a href="/dashboard">Dashboard</a></li>
```

Comment:
Why was this changed? Why link to `/dashboard` if the user will be redirected to `/`?